app/models/speaker.js

```diff
@@ -32,6 +33,12 @@ export default ModelBase.extend({
   gender             : attr('string'),
   heardFrom          : attr('string'),
 
+  segmentedLinkWebsite  : computedSegmentedLink.bind(this)('website'),
+  segmentedLinkTwitter  : computedSegmentedLink.bind(this)('twitter'),
```

Comment:
role back any changes to social media inputs for now. Strictly make changes to the website input.